Project.toml

```diff
@@ -51,6 +52,7 @@ ProgressLogging = "0.1"
 Reexport = "0.2, 1.0"
 Requires = "1.0"
 SciMLBase = "1.79.0, 2"
+SymbolicIndexingInterface = "0.3"
```

Comment:
This seems to not get resolved

Reply:
It's not released yet
https://github.com/SciML/SymbolicIndexingInterface.jl/pull/12